ui/lib/kv/addon/components/kv-page-header.hbs

```diff
@@ -35,7 +38,7 @@
 {{/if}}
 
 {{#if (or (has-block "toolbarFilters") (has-block "toolbarActions"))}}
-  <Toolbar aria-label="menu items for managing {{or @mountName @pageTitle}}">
+  <Toolbar aria-label="menu items for managing {{or @mountName @secretPath @pageTitle}}">
```

Comment:
My concern here would be, if in the future, someone uses this component with both a `secretPath` and a `pageTitle` the behavior would be unexpected. It seems to me that the design of this is that they are mutually exclusive: you can't have both, but only one. I checked the code and that seems to be the case. 

Some thoughts:
1. Create a kv-page-header.js file to have an assertion that you can't have both assigned. 
2. You could just stick with one property but then this else if block doesn't work. 
3. ... 🤔 maybe it's not a concern. 

Note: there are still some use cases of @pageTitle={{@path}} and if we go with the current logic those should be changed to @secretPath={{@path}}. Example [here](https://github.com/hashicorp/vault/blob/main/ui/lib/kv/addon/components/page/secret/metadata/version-history.hbs#L6).

Reply:
I think I like option 1 only because it also allows us to add documentation. But you've already gone above and beyond for a simple request so I'll make this comment non-blocking.

Reply:
This is just for the aria-label and so it follows the same logic flow for the header above, since they're mutually exclusive and you can only have one header text 😅 . I think the conditional above makes that pretty clear 🤔